tests/springshare/test_springshare.py

```diff
@@ -0,0 +1,55 @@
+import transmogrifier.models as timdex
+from transmogrifier.helpers import parse_xml_records
+from transmogrifier.sources.springshare import SpringshareOaiDc
+
+FIXTURES_PREFIX = "tests/fixtures/oai_dc/springshare"
+
+
+def test_springshare_get_dates_valid():
+    """
+    Test for valid date parsing
+    """
+
+    input_records = parse_xml_records(f"{FIXTURES_PREFIX}/springshare_valid_dates.xml")
+    transformer_instance = SpringshareOaiDc("libguides", input_records)
+
+    # asser valid dates
+    for xml in transformer_instance.input_records:
```

Comment:
Not sure this would work, given there are multiple `<record>` in the fixture.  The idea was that each would be tested, and because invalid, would throw the same invalid behavior.  If I'm reading correctly, the above would only process the _first_ `<record>` from the fixture.

Reply:
My bad, I mistakenly thought it was just testing 1 record, please disregard